src/index.html

```diff
@@ -4,9 +4,363 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div>
+        <a href="index.html" class="header__logo">
+          <img
+          class="header__logo-img"
+          src="images/logo.png"
+          alt="LOGO"
+          >
```

Comment:
```suggestion
          <img
            class="header__logo-img"
            src="images/logo.png"
            alt="LOGO"
          >
```
Не забывай про отступы